_studio/mfx_lib/CMakeLists.txt

```diff
@@ -137,6 +137,11 @@ foreach( prefix ${MSDK_STUDIO_ROOT}/shared/src )
     ${prefix}/mfx_mfe_adapter.cpp
   )
 endforeach()
+foreach( prefix ${MSDK_LIB_ROOT}/genx/field_copy/src )
```

Comment:
Android does not use cmake. Thus, this is unrelated change. Could you, please, extract it to the separate commit?

Reply:
done